packages/ui/src/components/MultixIdenticon.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import Tooltip from '@mui/material/Tooltip'
+import { ICON_SIZE, ICON_THEME } from '../constants'
+import Identicon from '@polkadot/react-identicon'
+import { IconButton } from '@mui/material'
+import { IconTheme } from '@polkadot/react-identicon/types'
+
+const DEFAULT_PLACEMENT = 'top'
+const DEFAULT_TITLE = 'Copied'
+const DEFAULT_AUTO_HIDE_DURATION = 2000
+
+interface Props {
+  value?: string
+  theme?: IconTheme
+  size?: number
+  className?: string
+}
+const MultixIdenticon = ({ value, theme = ICON_THEME, size = ICON_SIZE, className }: Props) => {
+  const [open, setOpen] = React.useState(false)
+  const handleTooltipClose = () => setOpen(false)
+  const handleTooltipOpen = () => setOpen(true)
+
+  return (
+    <Tooltip
+      PopperProps={{ disablePortal: true }}
+      onClose={handleTooltipClose}
+      open={open}
+      title={DEFAULT_TITLE}
+      placement={DEFAULT_PLACEMENT}
+      leaveDelay={DEFAULT_AUTO_HIDE_DURATION}
+      arrow
+    >
+      <IconButton onClick={handleTooltipOpen}>
```

Comment:
Is this needed or can we have the `onClick` handler on the Identicon directly?

Reply:
I'd appreciate an answer to this question

Reply:
Hey @Tbaut  sorry i guess i missed this.
So if we are using tooltip on `Icon` then we have to create a button of that `Icon` and need a wrapper around (using `<IconButton>` ) to use `Tooltip` but for Button there is not need of any wrapper like below: (Ref: https://mui.com/material-ui/react-tooltip/)

![image](https://github.com/ChainSafe/Multix/assets/61174028/1d0c8d7f-971b-44b3-8db5-041401d47e0b)

About using onClick() directly into Identicon we can do that. Though the method will change to onCopy() but works fine.

![image](https://github.com/ChainSafe/Multix/assets/61174028/27585263-6511-4176-b83c-c5ca2da2e6f7)

Let me know if we should change that. Thanks!!

Reply:
Yes, let's use `onCopy` then. No need to wrap it.

Reply:
Sorry but we have to use `<IconButton>` wrapper even if we use onCopy() handler. Because `Identicon` is an icon type not a button and `<Tooltip>` understands button only. I have committed the changes please check. Thanks!!

Reply:
The current state of the PR is not acceptable, it breaks the UI
![image](https://github.com/ChainSafe/Multix/assets/33178835/e33465ae-ab8e-4f34-aed9-f45fe972e4d3)

Reply:
You can change this IconButton by a div. Also please make sure the tooltip is on 1 line.
![image](https://github.com/ChainSafe/Multix/assets/33178835/bb2ebd77-49dc-4c2a-95c0-63a84f7843ea)

Reply:
Please check now its using same style as before. Thanks!!

[screen-recorder-mon-may-22-2023-16-34-34.webm](https://github.com/ChainSafe/Multix/assets/61174028/80aa466c-29a5-4083-976c-0f8bc88ef5bc)